iOSClient/More/NCMore.swift

```diff
@@ -97,35 +111,43 @@ class NCMore: UIViewController, UITableViewDelegate, UITableViewDataSource {
         labelQuotaExternalSite.text = ""
         progressQuota.progressTintColor = NCBrandColor.shared.brandElement
 
-        // ITEM : Transfer
-        item = NKExternalSite()
-        item.name = "_transfers_"
-        item.icon = "arrow.left.arrow.right"
-        item.url = "segueTransfers"
-        item.order = 10
-        functionMenu.append(item)
+//        // ITEM : Transfer
+//        item = NKExternalSite()
+//        item.name = "_transfers_"
+//        item.icon = "arrow.left.arrow.right"
+//        item.url = "segueTransfers"
+//        item.order = 10
+//        functionMenu.append(item)
 
         // ITEM : Recent
         item = NKExternalSite()
         item.name = "_recent_"
-        item.icon = "clock.arrow.circlepath"
+        item.icon = "recent"
         item.url = "segueRecent"
         item.order = 20
         functionMenu.append(item)
 
-        // ITEM : Activity
+        // ITEM : Notification
```

Comment:
Instead of modifying the existing Activity segue create separate segue for Notification.